### ISSUE TITLE
Opencast Integration

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/opencast-studio#272]
When we integrate Studio into Opencast, we do not need most of the configuration. The integration is indicated by the availability of the file `config.json` which is located in the same directory Studio is served from.

### config.json

```json
{
 "server": "<base url to use for ingest>",
 "workflow": "<workflow identifier to use for ingests>",
 "aurhenticated": "<true if no auth is required and credential input fiels should be hidden>"
}
```

### `?config=` URL parameter

**Optionally**, the configuration may also be provided via URL parameter (URL encoded JSON or base64 encoded JSON) which takes precedence over the configuration file. The parameter should have the same content.

**Comment by LukasKalbertodt** [comment]
I guess we should also remove `sentry` in the version integrated into Opencast?

**Comment by LukasKalbertodt** [comment]
Studio shipped with Opencast 8.2 and will be included in all future releases. There are surely still problems with the integration, but these should be tracked by new issues. So let's close this issue as done!
